mapswipe_workers/mapswipe_workers/generate_stats/project_stats.py

```diff
@@ -304,7 +312,22 @@ def get_agg_results_by_task_id(
 
     # add task geometry using left join
     agg_results_df = results_by_task_id_df.merge(
```

Comment:
same as above it might be good to check the project type first and then join the columns.

Reply:
or you just add a check here if the additional columns actually exist in the `task_df` dataframe. Then we don't need the project type information.

Reply:
would a simple df.dropna(how='all', axis=1, inplace=True) suffice? -> drops all empty columns

Reply:
yes, that would be a good option. I think ideal would be that we only drop the additional columns that are empty. And always keep the original ones, e.g. 1_count, 2_count etc.